charts/pega/charts/installer/values.yaml

```diff
@@ -34,6 +34,9 @@ bypassLoadEngineClasses: "false"
 bypassLoadAssembledClasses: "false"
 # If 'true', Helm will wait for the install or upgrade to finish, and only succeed if the job completes without error.
 waitForJobCompletion: "false"
+# If 'true', a PodDisruptionBudget will be created for the job's pod, preventing unexpected disruption
+#    ie from Cluster scaledown triggering node drain on node running job's pod
+preventInterruption: "false"
```

Comment:
What are the use cases for not having it enabled by default?

Reply:
Short answer, none that I can think of. I would think due to the nature of these jobs -- runs only once, can't really resume or retry -- that we'd want to prevent interruptions always. But was also wary of introducing the change in behavior, so instead chose to make it opt-in that we can set in PegaCloud, and optional for CMC